src/table/Exposure.cc

```diff
@@ -121,15 +122,23 @@ struct PersistenceHelper {
     // Read psf, wcs, etc. from an archive to an ExposureRecord
     void readRecord(
         BaseRecord const & input, ExposureRecord & output,
-        SchemaMapper const & mapper, io::InputArchive const & archive
+        SchemaMapper const & mapper, io::InputArchive const & archive,
+        int tableVersion
     ) const {
         output.assign(input, mapper);
         output.setPsf(archive.get<detection::Psf>(input.get(psf)));
         output.setWcs(archive.get<image::Wcs>(input.get(wcs)));
         output.setCalib(archive.get<image::Calib>(input.get(calib)));
         output.setApCorrMap(archive.get<image::ApCorrMap>(input.get(apCorrMap)));
         output.setValidPolygon(archive.get<geom::polygon::Polygon>(input.get(validPolygon)));
-        output.setVisitInfo(archive.get<image::VisitInfo>(input.get(visitInfo)));
+        if (tableVersion > 1) {
+            output.setVisitInfo(archive.get<image::VisitInfo>(input.get(visitInfo)));
+        } else {
+            // VisitInfo not available; set to invalid values
+            daf::base::PropertySet md;
+            auto nullVisitInfoPtr = std::make_shared<image::VisitInfo>(md);
+            output.setVisitInfo(nullVisitInfoPtr);
+        }
```

Comment:
If you simply do nothing, it should already be set to an empty pointer.  If you want to make it explicit (for readers of the code), you could do:
```
output.setVisitInfo(nullptr);
```
I'm ambivalent about which is better.